dss/util/tracing.py

```diff
@@ -19,6 +21,26 @@
     patched = True
 
 
+class XrayLoggerFilter(logging.Filter):
+    def filter(self, record):
+        if record.msg == MISSING_SEGMENT_MSG:
+            return False
+        try:
+            entity = xray_recorder.get_trace_entity()
+        except RecursionError:
+            return True
+        else:
+            record.xray_trace_id = entity.trace_id if entity else ""
+            record.msg = f"{record.xray_trace_id}\t{record.msg}"
```

Comment:
How about just a space since I'm pretty sure all other column breaks are also spaces?

Reply:
They are actually `tabs`, so i'm just sticking with consistency for now.

Reply:
kk